superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -449,7 +456,7 @@ const ResultSet = ({
               <ButtonGroup>
                 <Button
                   buttonSize="small"
-                  className="m-r-5"
```

Comment:
If this PR doesn't do it, we can probably do a sweep to get rid of these weird little utility classes soon.

Reply:
Yup, out of scope of this PR